src/env/mod.rs

```diff
@@ -75,13 +80,13 @@ pub(crate) fn set_remaining_points<T: WasmerEnv>(
     Ok(())
 }
 
-pub(crate) fn sub_remaining_gas<T: WasmerEnv>(env: &impl MassaEnv<T>, gas: u64) -> ABIResult<()> {
+pub(crate) fn sub_remaining_gas<T>(env: &impl MassaEnv<T>, store: &mut impl AsStoreMut, gas: u64) -> ABIResult<()> {
     if cfg!(feature = "gas_calibration") {
         return Ok(());
     }
-    let remaining_gas = get_remaining_points(env)?;
+    let remaining_gas = get_remaining_points(env, store)?;
     if let Some(remaining_gas) = remaining_gas.checked_sub(gas) {
-        set_remaining_points(env, remaining_gas)?;
+        set_remaining_points(env, store,remaining_gas)?;
```

Comment:
I think a little cargo fmt run is needed !

Reply:
done